index.html

```diff
@@ -224,5 +167,40 @@ <h2 class="meetup-date">24 nov</h2>
     </div>
   </footer>
 </body>
+<script>
+fetch('https://api.github.com/repos/devfsa/meetups/issues/3/comments')
+  .then((response)=> {
+      response.json().then((data)=> {
+        data = data.slice(0,4)
```

Comment:
U're slicing the comments sorted by creation, not reactions.
```suggestion
        // will wait all fetch from comments' reactions
        Promise.all(
          data.map(d => ({
            url: d.html_url,
            user: d.user,
            body: d.body,
            title: body.split('\r\n')[0].replace(/\*\*/g, '')
          })
          // will transform array of data into array of promises that returns data
          .map(keynote => (
            fetch(`${keynote.url}/reactions`)
              .then(res => res.json())
              .then(reactions => Object.assign({}, keynote, { reactions: reactions || [] }) )
              .catch(err => {
                console.log(`Error fetching reactions of ${keynote.title} // ${keynote.url}`)
                console.error(err);
                return Object.assign({}, keynote, { reactions: [] });
              })
          ))
        ).then(keynotes => {
          // after fetching all comments' reactions, sort it and then slice it
          return keynotes
            .sort((k1, k2) => k1.reactions.length - k2.reactions.length)
            .slice(0, 4)
        });
```

Reply:
I was making these changes after @di3goleite's comment. Thank you for that.